styles/LoginDropdown.module.css

```diff
@@ -1,3 +1,7 @@
+.navLink:hover {
+  color:rgb(234, 185, 239) !important;
+  transform: scale(1.1); 
+}
```

Comment:
Tbh I do not like the hover...please check out https://www.amupedia.com/. see hover effect